go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -133,12 +133,14 @@ func (nt *nodeTracker) Start(nodeInformer coreinformers.NodeInformer) (cache.Res
 			// - the name of the node (very rare) has changed
 			// - the `host-cidrs` annotation changed
 			// - node changes its zone
+			// - node changes its encap ip
 			// . No need to trigger update for any other field change.
 			if util.NodeSubnetAnnotationChanged(oldObj, newObj) ||
 				util.NodeL3GatewayAnnotationChanged(oldObj, newObj) ||
 				oldObj.Name != newObj.Name ||
 				util.NodeHostCIDRsAnnotationChanged(oldObj, newObj) ||
 				util.NodeZoneAnnotationChanged(oldObj, newObj) ||
+				util.NodeEncapIpAnnotationChanged(oldObj, newObj) ||
```

Comment:
Confused here. Is this annotation used for anything related to services?

Reply:
Sorry that was a left over experiment to get the encap-ip to trigger ovnkube-controller to update the chassis info in sbdb. The changes in `go-controller/pkg/ovn/default_network_controller.go` is the proper one.